e2e_test.go

```diff
@@ -74,6 +74,8 @@ func TestE2e(t *testing.T) {
 		})
 
 		if len(manualRemediations) > 0 {
+			// Wait some time after MachineConfigPool is ready to apply manual remediation
+			time.Sleep(60 * time.Second)
```

Comment:
It might be good to break this change out from the IdP change below.

Will this increase e2e test run times significantly? If so, and we decide to revert it later, I wouldn't want to lose the IdP logs below because we reverted all this in a single patch. That said, I'm not sure how legitimate my timing question is, so I'll see what other think about this approach.

Reply:
I had them in two different commits. This will only add 1 minute to the test.